ecommerce-ui/src/VacationRental.js

```diff
@@ -0,0 +1,59 @@
+import React  from 'react';
+import StarRatingComponent from 'react-star-rating-component';
+import PropTypes from 'prop-types';
+
+export default class VacationRental extends React.Component {
+
+render() {
+    const idx = this.props.idx;
+    const style = { display: this.props.listing.host.isSuperhost  ? 'block' : 'none'};
+    const rental = this.props.listing;
+     return(
+         
+        <div className="listing">
+        <div className="listing_image">
+        <img className="rental_image" src={rental.image} alt="rental-img"/>
+        </div>
+        <div className="listing_text">
+        <div>
+
+        </div>
+        <div className="listing_type">
+        <p className="a">{rental.houseType} . {rental.location.city} </p>
+        </div>
+       
+        <div className="listing_name">
+         <p className="b"> {this.props.listing.title}</p>
+        </div>
+       
+        <div className="listing_price">
+        <p className="c"> {rental.payment.cost} per night . {rental.payment.description}</p>
+       </div>
+
+       <div className="listing_rating">
+
+       <StarRatingComponent 
+          name="ratings" 
+          value={rental.rating.stars}
+        />&nbsp;
+        <span className="c">{rental.rating.reviews}</span> 
```

Comment:
Nit: In the future, try to use more meaningful class names so it's easier to change + maintain later.